contrib/lang/scala/packages.el

```diff
@@ -47,6 +47,78 @@ which require an initialization must be listed explicitly in the list.")
         (kbd "n") 'forward-button
         (kbd "N") 'backward-button)
 
+      (defun ensime-gen-and-reload()
+        (interactive)
+        (progn
+          (sbt-command "gen-ensime")
+          (ensime-shutdown)
+          (ensime))
+        )
+      
+      (evil-leader/set-key-for-mode 'scala-mode
+        "mg"     'ensime-edit-definition
```

Comment:
This is a temporary binding, but the function should be included in the bindings